File: olderFiles/stang/objective_c/first/first/ViewController.m
//
//  ViewController.m
//  first
//
//  Created by SG on 2016-09-14.
//  Copyright © 2016 SG. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@property (nonatomic, weak) IBOutlet UILabel *messageLabel;
@property (nonatomic, weak) IBOutlet UITextField *inputField;

@end

@implementation ViewController

- (IBAction) Enter {
    NSString *yourName = self.inputField.text;
    NSString *myName = @"Mark";
    NSString *msg;
    
    
    if ([yourName length] == 0){
    yourName = @"HelloWorld";
    }
    
    if ([yourName isEqualToString:myName]){
        msg = [NSString stringWithFormat:@"Hello %@, we have same name",yourName];
    } else {
        msg = [NSString stringWithFormat:@"Hello %@", yourName];
    
    }
    
    self.messageLabel.text = msg;
}

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 401_2016_2/swift_examples/properties/properties/ViewController.swift
//
//  ViewController.swift
//  properties
//
//  Created by Luiz on 2016-10-11.
//  Copyright © 2016 Ideia do Luiz. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    var count = 0
    @IBOutlet weak var ssasa: UILabel!

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func ddsds(_ sender: AnyObject) {
        self.count += 1
        self.ssasa.text = String(self.count)
        
    }

}

